neomodel/match.py

```diff
@@ -496,16 +504,16 @@ def _execute(self, lazy=False):
             # inject id = into ast
             self._ast['return'] = 'id({})'.format(self._ast['return'])
         query = self.build_query()
-        results, _ = db.cypher_query(query, self._query_params, resolve_objects=True)            
+        results, _ = db.cypher_query(query, self._query_params, resolve_objects=True)
```

Comment:
Not sure why this whitespace was marked as a change..?

Reply:
No idea. I just removed some whitespace because it was bothering my eyes but that's all it was.